apps/app_rpt.c

```diff
@@ -5095,6 +5186,15 @@ static void *rpt(void *this)
 		/* Main TX control */
 
 		/* let telemetry transmit anyway (regardless of timeout) */
+
+		/* Fixme note: is is somewhat undesired as we really only want to
```

Comment:
Not sure this will get 100% of the "bad" telemetry, but maybe we could / should flush the telemetry (using `flush_telem(myrpt)` ) before sending the timeout notification here:
https://github.com/AllStarLink/app_rpt/blob/45c573286166bf4a0e9a1c3540bffb9a30e67cde/apps/app_rpt.c#L5001-L5007

Reply:
Do we need a "new flag"? we have the mode stored in there.

Reply:
Other telemetry messages in the future could need to override the timeout. Save this question for the pull request when I create it We can revisit it then.